dnnsoftware/Dnn.AdminExperience.Extensions
# Adds back the UI to edit the Host Email address

# Testing steps:
- Go to Settings -> Servers -> Server Settings -> SMTP Server
- Change the host email and save
- Go to Settings -> SQL Console
- Enter the following query: 
```SQL
SELECT SettingValue FROM {databaseOwner}{objectQualifier}HostSettings
WHERE SettingName = 'HostEmail'
```
- Make sure the returned email is the one entered in the UI

# Context
The host email is used in a couple places in the platform, see https://github.com/dnnsoftware/Dnn.Platform/search?q=hostemail&unscoped_q=hostemail
It can also be used by module developers to send a message to the person responsible for hosting the possibly multiple sites.
Since Dnn 9 and the persona bar came out, the only way to change this host email was to edit the database directly.

closes #604 

This is my first pull request for the persona bar, if I did anything wrong or not following best practices, please comment.
